messages/eslintrc-plugins.js

```diff
@@ -5,9 +5,14 @@ module.exports = function({ plugins }) {
     const isArrayOfStrings = typeof plugins[0] === "string";
 
     return `
-A config object has a "plugins" key defined as an array${isArrayOfStrings ? " of strings" : ""}.
+A config object has a "plugins" key defined as an array${isArrayOfStrings ? " of strings" : ""}. It
+looks something like this:
```

Comment:
May be we can write it like this?
```suggestion
A config object has a "plugins" key defined as an array${isArrayOfStrings ? " of strings" : ""}.
It looks something like this:
```
OR

```suggestion
A config object has a "plugins" key defined as an array${isArrayOfStrings ? " of strings" : ""}. It looks something like this:
```